### PR TITLE
add preprocess and argmax to export process

### PR DESCRIPTION
**Pull request by wuyefeilin**

### CODE REVIEW

File: pdseg/models/model_builder.py [hunk 1]
```diff
@@ -140,8 +140,23 @@ def build_model(main_prog, start_prog, phase=ModelPhase.TRAIN):
 
     with fluid.program_guard(main_prog, start_prog):
         with fluid.unique_name.guard():
-            image = fluid.layers.data(
-                name='image', shape=image_shape, dtype='float32')
+            if ModelPhase.is_predict(phase):
```
Comment (ZeyuChen) [inline pdseg/models/model_builder.py]: Please add comments on critical coding.
eg. sub mean and divide variance, need to tell users why add this op in Predict phrase

File: pdseg/models/model_builder.py [hunk 2]
```diff
@@ -217,7 +232,10 @@ def build_model(main_prog, start_prog, phase=ModelPhase.TRAIN):
                     logit = sigmoid_to_softmax(logit)
                 else:
                     logit = softmax(logit)
-                return image, logit
+                logit = fluid.layers.resize_bilinear(logit, out_shape=origin_shape, align_corners=False, align_mode=0)
+                logit = fluid.layers.transpose(logit, [0, 2, 3, 1])
+                logit = fluid.layers.argmax(logit, axis=3)
+                return origin_image, logit
 
             if class_num == 1:
```
Comment (ZeyuChen) [inline pdseg/models/model_builder.py]: 需要添加注释

File: pdseg/models/model_builder.py (hunk 1 above)
Comment (wuyefeilin) [reply pdseg/models/model_builder.py]: 已增加注释

File: pdseg/models/model_builder.py (hunk 2 above)
Comment (wuyefeilin) [reply pdseg/models/model_builder.py]: 已增加注释

**Pull request merged**